.github/actions/smart-ci/requirements.txt

```diff
@@ -0,0 +1,3 @@
+PyGithub==1.59
```

Comment:
maybe it would be better to use ~=? e.g. PyGithub~=1.59.0

Reply:
Looks like formatting has gone bad :) Could you wrap your suggestion to code block?

Reply:
Discussed offline, applied suggested change